pkg/queue/cluster_queue_impl_test.go

```diff
@@ -36,7 +37,9 @@ const (
 )
 
 func Test_PushOrUpdate(t *testing.T) {
-	cq := newClusterQueueImpl(keyFunc, queueOrdering)
+	cq := newClusterQueueImpl(keyFunc, queueOrderingFunc(workload.Ordering{
```

Comment:
nit: maybe create a var in the test so that we don't repeat it?
```
defaultQueueOrdering := queueOrderingFunc(workload.Ordering{
		PodsReadyRequeuingTimestamp: config.Eviction,
	})
```

Reply:
Still unresolved

Reply:
Added